csm_web/frontend/src/components/Policies.tsx

```diff
@@ -0,0 +1,498 @@
+        "designed by experienced mentors. No homework or project help " +
+        " will ever be provided. However, if your entire group agrees, " +
+        "you can bring in questions from discussion worksheets, guerrilla " +
+        "sections, etc. In this case, please give your mentor a head up " +
+        "so that they can prepare.",
+      i4_1:
+        "For each class that you are a CSM student for, you may receive " +
+        "one P/NP unit (see below). You will receive these unit(s) if you " +
+        "attend section (same time, same place) each week. ",
+      i4_b: "Sign up for a section you are confident that you can make " + "it for the entire semester. ",
+      i4_2: "You can have two unexcused absences over the course of the semester. ",
+      i5:
+        "We will release a couple of surveys—one midway through the " +
+        "semester, and one at the end of the semester—to collect feedback " +
+        "on your mentor and your experience with CSM. We want to become " +
+        "better and better with each passing semester, and we can only do " +
+        "that with your assistance. These surveys are mandatory if you are " +
+        "enrolled in a CSM section (see below), but we promise they will not " +
+        "take long to fill out."
+    }
+  },
+  communications: {
+    title: "Communications",
+    body: {
+      p1_1: "Worksheets can be found on the ",
+      p1_a: {
+        link: "https://scheduler.csmentors.org/",
+        text: "CSM Scheduler"
+      },
+      p1_2:
+        ' website under "Resources". Worksheets are publicly available, ' +
+        "even for students who are not enrolled in a CSM section.",
+      p2_1: "General questions regarding CSM can be directed to our email address, ",
+      p2_a: {
+        link: "mailto:mentors@berkeley.edu",
+        text: "mentors@berkeley.edu"
+      },
+      p2_2:
+        ". For communications specific to your CSM course, some courses " +
+        "will have a CSM Piazza, which will be titled CSM [Course Number]. " +
+        "Your mentor will be your primary contact for questions regarding your section."
+    }
+  },
+  affinitySections: {
+    title: "Affinity Sections",
+    body: {
+      p1:
+        "We offer several sections geared towards supporting students from " +
+        "backgrounds underrepresented or under-resourced in EECS and STEM. " +
+        "These include:",
+      i1_b: "Affinity Sections",
+      i1_2:
+        " - These are sections designed for students who affiliate with " +
+        "identities and communities underrepresented in the Berkeley EECS " +
+        "community. Types of affinity sections that we have offered " +
+        "previously, or hope to offer, include:",
+      i1_i1: "1.1 Woman affinity sections",
+      i1_i2: "1.2 LGBTQ+ affinity sections",
+      i1_i3: "1.3 Hispanic/Latinx affinity sections",
+      i1_i4: "1.4 Transfer sections",
```

Comment:
The numberings here should probably be omitted, and instead replaced by unordered lists in the HTML

Reply:
Resolved, removed numbers and added items to an unordered list!